wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginMeta.java

```diff
@@ -109,4 +118,85 @@ public PluginMeta attributes(Map<String, String> attributes) {
     this.attributes = attributes;
     return this;
   }
+
+  public PluginMeta category(String category) {
+    this.category = category;
+    return this;
+  }
+
+  public String getCategory() {
+    return category;
+  }
+
+  public PluginMeta publisher(String publisher) {
+    this.publisher = publisher;
+    return this;
+  }
+
+  public String getPublisher() {
+    return publisher;
+  }
+
+  public PluginMeta repository(String repository) {
+    this.repository = repository;
+    return this;
+  }
+
+  public String getRepository() {
+    return repository;
+  }
+
+  public PluginMeta tags(String tags) {
+    this.tags = tags;
+    return this;
+  }
+
+  public String getTags() {
+    return tags;
+  }
+
+  public PluginMeta mediaImage(String mediaImage) {
+    this.mediaImage = mediaImage;
+    return this;
+  }
+
+  public String getMediaImage() {
+    return mediaImage;
+  }
+
+  public PluginMeta mediaVideo(String mediaVideo) {
+    this.mediaVideo = mediaVideo;
+    return this;
+  }
+
+  public String getediaVideo() {
```

Comment:
I guess there is typo in this method name

Reply:
ty